app/models/organization.rb

```diff
@@ -5,7 +5,9 @@ class Organization < ActiveRecord::Base
   enum entity_type: [:association, :federation, :lobby]
 
   validates :inscription_reference, uniqueness: true, allow_blank: true, allow_nil: true
-  validates :name, :user, :category_id, presence: true
+  validates :name, :category_id, presence: true
+  # PublicOrganizationImporter gets organizations without related user
```

Comment:
Why do we keep a comment on the code? Is this something that should be fixed afterwards because right not its not possible? If so please create an issue describing what's needed, comments on code get forgotten normally

Reply:
Its from the other branch I mentiones above (I messed things up a bit with a merge)